src/auto/display.rs

```diff
@@ -257,9 +260,11 @@ impl Display {
         }
     }
 
-    //pub fn store_clipboard(&self, clipboard_window: &Window, time_: u32, targets: /*Ignored*/&[&Atom], n_targets: i32) {
-    //    unsafe { TODO: call ffi::gdk_display_store_clipboard() }
-    //}
+    pub fn store_clipboard(&self, clipboard_window: &Window, time_: u32, targets: &[&Atom], n_targets: i32) {
+        unsafe {
+            ffi::gdk_display_store_clipboard(self.to_glib_none().0, clipboard_window.to_glib_none().0, time_, targets.to_glib_none().0, n_targets);
```

Comment:
I think we want this to be `&[Atom]` since it's an input. The natural way to generate this will be to create a `Vec<Atom>` and pass a slice to the function. If the type is a slice of references, it's going to take an extra conversion to use this in the common case I think. It also shouldn't have the `target_length` argument, so I'll need to manually implement this.